syntax = "proto3";

package user;

service UserService {
  rpc GetUserById (UserIdRequest) returns (UserResponse);
}

message UserIdRequest {
  int32 id = 1;
}

message UserResponse {
  int32 id = 1;
  string username = 2;
  string role = 3;
}
